packages/excalidraw/types.ts

```diff
@@ -258,8 +266,13 @@ export interface AppState {
   };
   editingFrame: string | null;
   elementsToHighlight: NonDeleted<ExcalidrawElement>[] | null;
-  // element being edited, but not necessarily added to elements array yet
-  // (e.g. text element when typing into the input)
+  /**
+   * currently set for:
+   * - text elements while in wysiwyg
+   * - newly created linear elements while in line editor (not set for existing
+   *   elements in line editor)
+   * - and new images while being placed on canvas
+   */
```

Comment:
added docs for `editingElement`. I'd say this one is ripe for refactor as well as it doesn't make much sense.